src/WarcraftLegacies.Source/Quests/Naga/QuestNajentus.cs

```diff
@@ -20,18 +24,23 @@ public sealed class QuestNajentus : QuestData
       AddObjective(new ObjectiveControlPoint(ControlPointManager.Instance.GetFromUnitType(Constants.UNIT_N00P_THE_ABYSS_25GOLD_MIN)));
       AddObjective(new ObjectiveControlPoint(ControlPointManager.Instance.GetFromUnitType(Constants.UNIT_N028_DROWNED_REACHES_10GOLD_MIN)));
       ResearchId = Constants.UPGRADE_R08W_QUEST_COMPLETED_LORD_OF_THE_DEPTHS;
+      Required = true;
     }
 
+    /// <inheritdoc/>
     protected override string CompletionPopup =>
-      "The sea belong to Lord Illidan!";
+      "Now that Nazjatar is under Illidan's control, Lady Vashj's champion Lord Naj'entus has joined Illidan's forces!";
 
-    protected override string RewardDescription => "The hero Naj'entus will join you!";
+    /// <inheritdoc/>
+    protected override string RewardDescription => "The hero Naj'entus is now trainable at the altar.";
```

Comment:
Can we mention the actual name of the Altar where Naj'entus can be trained from? Refer to `QuestJergosh.cs` line 32 for an example of this.